build: Include native_X.mk before X.mk

hebasto opened the pull request:
Unfortunately, our depends build system still lacks 100% reliability.

On master (57982f419e36d0023c83af2dd0d683ca3160dc2a):
```
$ make --no-print-directory -C depends print-capnp_version MULTIPROCESS=1
capnp_version=
```

This PR fixes this issue:
```
$ make --no-print-directory -C depends print-capnp_version MULTIPROCESS=1
capnp_version=0.7.0
```

This PR split off from bitcoin/bitcoin#22552, bitcoin/bitcoin#22555 and bitcoin/bitcoin#22708.

hebasto committed:
build: Include native_X.mk before X.mk

This change fixes some weirdness like `make print-capnp_version` output
in depends.

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
481a54ccede27a02e51810987ddbca63145e36d0bddc2556c418a2276a295518  guix-build-980a9df6a875/output/aarch64-linux-gnu/SHA256SUMS.part
ae3456e4645528beadcb6685e362fc4576994f26022fdaf56974f51854f75aa4  guix-build-980a9df6a875/output/aarch64-linux-gnu/bitcoin-980a9df6a875-aarch64-linux-gnu-debug.tar.gz
e5d25c0b6f386429b78163c5f5047df0a2829614f5b00c22ff48642229056f18  guix-build-980a9df6a875/output/aarch64-linux-gnu/bitcoin-980a9df6a875-aarch64-linux-gnu.tar.gz
e5d1858a52d46fb736b994095a5288d794460794ad4adc2f2224c6d179404986  guix-build-980a9df6a875/output/arm-linux-gnueabihf/SHA256SUMS.part
841f1414fba31de57955b6b06797d387925ad4a40bcdefefae4ba979a21c02a6  guix-build-980a9df6a875/output/arm-linux-gnueabihf/bitcoin-980a9df6a875-arm-linux-gnueabihf-debug.tar.gz
836167ae9a54284eec1b97e2ea50b46d650cf268902f15783e5e714790a5035c  guix-build-980a9df6a875/output/arm-linux-gnueabihf/bitcoin-980a9df6a875-arm-linux-gnueabihf.tar.gz
a705377c3efd26f3d7b186eeb3234677f24f646106faf0dee5dc6e0e824fc1be  guix-build-980a9df6a875/output/dist-archive/bitcoin-980a9df6a875.tar.gz
cae79768ce7a82a59a09644c5bda461705a2a482bab599688c4cbfe8121efcb8  guix-build-980a9df6a875/output/powerpc64-linux-gnu/SHA256SUMS.part
d3ce77413be5c62359e5eb7ec3d21f4c7b7e14179ea72bca9f93ac7a54ab4f76  guix-build-980a9df6a875/output/powerpc64-linux-gnu/bitcoin-980a9df6a875-powerpc64-linux-gnu-debug.tar.gz
55dd6a475747267551cb1c0cf7422dd00c8336c63c0d992fbd6eb77b9670dfb5  guix-build-980a9df6a875/output/powerpc64-linux-gnu/bitcoin-980a9df6a875-powerpc64-linux-gnu.tar.gz
6e3f9824b98e7eec451c05f7f6657fc70b6108250a38d0f68bc3a33423dfc87f  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/SHA256SUMS.part
5ab13aa207211fc4eb2f541840d7ab545ada48461985e9eaf0983c5926009b1f  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/bitcoin-980a9df6a875-powerpc64le-linux-gnu-debug.tar.gz
e377f158e06bf33a504c008bfc501581d4867a6b953fefbc717a2f1b34e1942c  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/bitcoin-980a9df6a875-powerpc64le-linux-gnu.tar.gz
99d12be30ac9a963b0db8a0620207f1e1637297acf80919bdc139e53133591e5  guix-build-980a9df6a875/output/riscv64-linux-gnu/SHA256SUMS.part
f2108cea5a06b2608a8adb5a0d7244153c2e4b5a2addda3e0ac8cf5a21efa16f  guix-build-980a9df6a875/output/riscv64-linux-gnu/bitcoin-980a9df6a875-riscv64-linux-gnu-debug.tar.gz
4321cd179d3d782b4bcd175a1d262d9c010e28009fbd459875b6c75de85ea068  guix-build-980a9df6a875/output/riscv64-linux-gnu/bitcoin-980a9df6a875-riscv64-linux-gnu.tar.gz
567f24f3ac1c073365b0e93dcdb55b4b2fa7a9b1dcf02d8236bcd441f5c2b4f1  guix-build-980a9df6a875/output/x86_64-apple-darwin19/SHA256SUMS.part
e6523ed9bbc750b373e49de1c1b7c676231eceded8d4dfb63d9a80c29434d731  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx-unsigned.dmg
b73f311addf54d92dc6cff763dc7dec20c0f974002e7af3dbe6ea8f1487aa998  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx-unsigned.tar.gz
ecf897cdef980374a01e31dd6e8d350c185caa29e8381b3f01b366c82e2a132c  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx64.tar.gz
2c81ade74732784b600cde78ee9c39611bfb7fe0ec1de506d3a097d4b925add7  guix-build-980a9df6a875/output/x86_64-linux-gnu/SHA256SUMS.part
bbc98e3dcd7bafa431a8f22e177d56bdb6c3dc5a2339acdfc19e12dd4b8be21c  guix-build-980a9df6a875/output/x86_64-linux-gnu/bitcoin-980a9df6a875-x86_64-linux-gnu-debug.tar.gz
3d7c83829722f59caa909c59a7742a07b49d621fda201495a5d23d152941231d  guix-build-980a9df6a875/output/x86_64-linux-gnu/bitcoin-980a9df6a875-x86_64-linux-gnu.tar.gz
ef2a09eed729af4e0ef36a4aa3ed52c24f5ce104f5bf7e27108b9e5acc3fbd07  guix-build-980a9df6a875/output/x86_64-w64-mingw32/SHA256SUMS.part
3bb35e271b14dee466d9bc62e36c9d16b902118a5b8a3733ac7aa677151f3a6a  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win-unsigned.tar.gz
d1e2a6fdbdace7be81818814460d0a28cf4a19a95a17939b3c9595cc48185b74  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64-debug.zip
4dde98b310b578f5144705d301dbc7a677ef4b93811d4231cbd5df5bdea98a3f  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64-setup-unsigned.exe
a2842b952ddf8f4e15710b34f140d2ceedd268b1476cabad2c4d30fd53a129a0  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64.zip
```

jarolrod commented:
Guix hashes, mine match @hebasto:

```
481a54ccede27a02e51810987ddbca63145e36d0bddc2556c418a2276a295518  guix-build-980a9df6a875/output/aarch64-linux-gnu/SHA256SUMS.part
ae3456e4645528beadcb6685e362fc4576994f26022fdaf56974f51854f75aa4  guix-build-980a9df6a875/output/aarch64-linux-gnu/bitcoin-980a9df6a875-aarch64-linux-gnu-debug.tar.gz
e5d25c0b6f386429b78163c5f5047df0a2829614f5b00c22ff48642229056f18  guix-build-980a9df6a875/output/aarch64-linux-gnu/bitcoin-980a9df6a875-aarch64-linux-gnu.tar.gz
e5d1858a52d46fb736b994095a5288d794460794ad4adc2f2224c6d179404986  guix-build-980a9df6a875/output/arm-linux-gnueabihf/SHA256SUMS.part
841f1414fba31de57955b6b06797d387925ad4a40bcdefefae4ba979a21c02a6  guix-build-980a9df6a875/output/arm-linux-gnueabihf/bitcoin-980a9df6a875-arm-linux-gnueabihf-debug.tar.gz
836167ae9a54284eec1b97e2ea50b46d650cf268902f15783e5e714790a5035c  guix-build-980a9df6a875/output/arm-linux-gnueabihf/bitcoin-980a9df6a875-arm-linux-gnueabihf.tar.gz
a705377c3efd26f3d7b186eeb3234677f24f646106faf0dee5dc6e0e824fc1be  guix-build-980a9df6a875/output/dist-archive/bitcoin-980a9df6a875.tar.gz
cae79768ce7a82a59a09644c5bda461705a2a482bab599688c4cbfe8121efcb8  guix-build-980a9df6a875/output/powerpc64-linux-gnu/SHA256SUMS.part
d3ce77413be5c62359e5eb7ec3d21f4c7b7e14179ea72bca9f93ac7a54ab4f76  guix-build-980a9df6a875/output/powerpc64-linux-gnu/bitcoin-980a9df6a875-powerpc64-linux-gnu-debug.tar.gz
55dd6a475747267551cb1c0cf7422dd00c8336c63c0d992fbd6eb77b9670dfb5  guix-build-980a9df6a875/output/powerpc64-linux-gnu/bitcoin-980a9df6a875-powerpc64-linux-gnu.tar.gz
6e3f9824b98e7eec451c05f7f6657fc70b6108250a38d0f68bc3a33423dfc87f  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/SHA256SUMS.part
5ab13aa207211fc4eb2f541840d7ab545ada48461985e9eaf0983c5926009b1f  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/bitcoin-980a9df6a875-powerpc64le-linux-gnu-debug.tar.gz
e377f158e06bf33a504c008bfc501581d4867a6b953fefbc717a2f1b34e1942c  guix-build-980a9df6a875/output/powerpc64le-linux-gnu/bitcoin-980a9df6a875-powerpc64le-linux-gnu.tar.gz
99d12be30ac9a963b0db8a0620207f1e1637297acf80919bdc139e53133591e5  guix-build-980a9df6a875/output/riscv64-linux-gnu/SHA256SUMS.part
f2108cea5a06b2608a8adb5a0d7244153c2e4b5a2addda3e0ac8cf5a21efa16f  guix-build-980a9df6a875/output/riscv64-linux-gnu/bitcoin-980a9df6a875-riscv64-linux-gnu-debug.tar.gz
4321cd179d3d782b4bcd175a1d262d9c010e28009fbd459875b6c75de85ea068  guix-build-980a9df6a875/output/riscv64-linux-gnu/bitcoin-980a9df6a875-riscv64-linux-gnu.tar.gz
567f24f3ac1c073365b0e93dcdb55b4b2fa7a9b1dcf02d8236bcd441f5c2b4f1  guix-build-980a9df6a875/output/x86_64-apple-darwin19/SHA256SUMS.part
e6523ed9bbc750b373e49de1c1b7c676231eceded8d4dfb63d9a80c29434d731  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx-unsigned.dmg
b73f311addf54d92dc6cff763dc7dec20c0f974002e7af3dbe6ea8f1487aa998  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx-unsigned.tar.gz
ecf897cdef980374a01e31dd6e8d350c185caa29e8381b3f01b366c82e2a132c  guix-build-980a9df6a875/output/x86_64-apple-darwin19/bitcoin-980a9df6a875-osx64.tar.gz
2c81ade74732784b600cde78ee9c39611bfb7fe0ec1de506d3a097d4b925add7  guix-build-980a9df6a875/output/x86_64-linux-gnu/SHA256SUMS.part
bbc98e3dcd7bafa431a8f22e177d56bdb6c3dc5a2339acdfc19e12dd4b8be21c  guix-build-980a9df6a875/output/x86_64-linux-gnu/bitcoin-980a9df6a875-x86_64-linux-gnu-debug.tar.gz
3d7c83829722f59caa909c59a7742a07b49d621fda201495a5d23d152941231d  guix-build-980a9df6a875/output/x86_64-linux-gnu/bitcoin-980a9df6a875-x86_64-linux-gnu.tar.gz
ef2a09eed729af4e0ef36a4aa3ed52c24f5ce104f5bf7e27108b9e5acc3fbd07  guix-build-980a9df6a875/output/x86_64-w64-mingw32/SHA256SUMS.part
3bb35e271b14dee466d9bc62e36c9d16b902118a5b8a3733ac7aa677151f3a6a  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win-unsigned.tar.gz
d1e2a6fdbdace7be81818814460d0a28cf4a19a95a17939b3c9595cc48185b74  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64-debug.zip
4dde98b310b578f5144705d301dbc7a677ef4b93811d4231cbd5df5bdea98a3f  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64-setup-unsigned.exe
a2842b952ddf8f4e15710b34f140d2ceedd268b1476cabad2c4d30fd53a129a0  guix-build-980a9df6a875/output/x86_64-w64-mingw32/bitcoin-980a9df6a875-win64.zip
```

fanquake commented on depends/funcs.mk:
```diff
@@ -264,7 +264,8 @@ $(foreach package,$(packages),$(eval $(package)_type=$(host_arch)_$(host_os)))
 
 #include package files
-$(foreach package,$(all_packages),$(eval include packages/$(package).mk))
+$(foreach package,$(native_packages),$(eval include packages/$(package).mk))
```
To match above?
```suggestion
$(foreach native_package,$(native_packages),$(eval include packages/$(native_package).mk))
```

hebasto replied:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23673#issuecomment-990706374).

fanquake reviewed (commented):
Concept ACK

This also fixes other weirdness like:
```bash
# master
gmake --no-print-directory -C depends print-capnp_dependencies MULTIPROCESS=1
capnp_dependencies=native_native_capnp

# this PR
gmake --no-print-directory -C depends print-capnp_dependencies MULTIPROCESS=1
capnp_dependencies=native_capnp
```

hebasto commented:
Updated 980a9df6a875b664f7d911fdb641f16d872011dd -> ef77549000ac46eededa3e4a89466024db111524 ([pr23673.01](https://github.com/hebasto/bitcoin/commits/pr23673.01) -> [pr23673.02](https://github.com/hebasto/bitcoin/commits/pr23673.02)):

- rebased
- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/23673#discussion_r766310034)

Zero-1729 reviewed (commented):
tACK ef77549000ac46eededa3e4a89466024db111524

Tested on macOS 12.0.1

Currently, on master (d5724b8c6a1f542edc0b02b6bd069121e2cd2542)

```sh
$ make --no-print-directory -C depends print-capnp_version MULTIPROCESS=1
capnp_version=
```

After this patch

```sh
$ make --no-print-directory -C depends print-capnp_version MULTIPROCESS=1
capnp_version=0.7.0
```

---

Also tested the weirdness fanquake pointed out above.

Master 

```sh
$ make --no-print-directory -C depends print-capnp_dependencies MULTIPROCESS=1
capnp_dependencies=native_native_capnp 
```

This patch

```sh
$ make --no-print-directory -C depends print-capnp_dependencies MULTIPROCESS=1 
capnp_dependencies=native_capnp 
```

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
3895e9c0877221d48d0c4c601a70f8568656190a98bc16852484c255909db98a  guix-build-ef77549000ac/output/aarch64-linux-gnu/SHA256SUMS.part
e08470526151885152d45f521309d50049639ff350ffbc6f3509ea86be0e391c  guix-build-ef77549000ac/output/aarch64-linux-gnu/bitcoin-ef77549000ac-aarch64-linux-gnu-debug.tar.gz
4a4f0ff43840acf9e563086c368f60cf7ead50d91357b0a6474cbfe5954870b4  guix-build-ef77549000ac/output/aarch64-linux-gnu/bitcoin-ef77549000ac-aarch64-linux-gnu.tar.gz
16a0145e0e787f16f26e68320074a52045338a90ee12ec83aaca6a99c0eb7681  guix-build-ef77549000ac/output/arm-linux-gnueabihf/SHA256SUMS.part
10a7aef5accffa67829ecd015ec8f52aa5907641cf4d60bcf29fd20e93418227  guix-build-ef77549000ac/output/arm-linux-gnueabihf/bitcoin-ef77549000ac-arm-linux-gnueabihf-debug.tar.gz
8ced201f710f4913fdff36061dbe01c7fc64a0dc97f92c00b22e512c09125fd9  guix-build-ef77549000ac/output/arm-linux-gnueabihf/bitcoin-ef77549000ac-arm-linux-gnueabihf.tar.gz
d50770610865e6978efc6e6c60e81358b88bf560872ba30bbe9c200cf2c4c618  guix-build-ef77549000ac/output/dist-archive/bitcoin-ef77549000ac.tar.gz
2fed92dc00eb5a96817ca955171fb1b9d18fe8d9ad67999568f1eac4e1e16279  guix-build-ef77549000ac/output/powerpc64-linux-gnu/SHA256SUMS.part
80b16d5cf1d4f5ee4aa8072b7c003f412236a0517feee5933afc768a8b21d4c2  guix-build-ef77549000ac/output/powerpc64-linux-gnu/bitcoin-ef77549000ac-powerpc64-linux-gnu-debug.tar.gz
b15ffbd4533bbc4f9411b749e0ca3938318369c4cfc61d7d28d03b09b3ac7373  guix-build-ef77549000ac/output/powerpc64-linux-gnu/bitcoin-ef77549000ac-powerpc64-linux-gnu.tar.gz
96665ee949e487170bf2cdb98caea3d530e8b922f9a9420ea222b0abda1a1754  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/SHA256SUMS.part
d7177210324ed92d6c35b615b8614b07b31f2a3ce543da55034e3bc680d24abb  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/bitcoin-ef77549000ac-powerpc64le-linux-gnu-debug.tar.gz
934cad3c42f1029e1f7f1f4b0a888b1d266ada67fa571b847a6d3c29ff894fa3  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/bitcoin-ef77549000ac-powerpc64le-linux-gnu.tar.gz
28df25c8dcd46177da706c43ebf272201aec36fdd93cbd73fade71e52851140f  guix-build-ef77549000ac/output/riscv64-linux-gnu/SHA256SUMS.part
bc72438937da78740fbc364d4c1102b3821d2c686ba4bc188001fcf7a3ca2b98  guix-build-ef77549000ac/output/riscv64-linux-gnu/bitcoin-ef77549000ac-riscv64-linux-gnu-debug.tar.gz
fbadeffc83e3dbb31a702e1406616469c5ad84aa70a34ca5b908fa680d7348f4  guix-build-ef77549000ac/output/riscv64-linux-gnu/bitcoin-ef77549000ac-riscv64-linux-gnu.tar.gz
acfdefec7ac276ba7bac88982a4200dc976263831a8581c87c00eef1a87a6c4f  guix-build-ef77549000ac/output/x86_64-apple-darwin/SHA256SUMS.part
237d290ef473736bf22e23f29fe277770497c39c9e36908b2816e3ce8dad2822  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx-unsigned.dmg
2bb6412ab793538f39aa73d5bb868a6b85da85d387eb672652d88ae7d7b18d26  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx-unsigned.tar.gz
605402a386e4444fe31fa70cc79096e4da446310616ad5b108afebfdec43fbf2  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx64.tar.gz
a5e752f3b796841a2edac7621c290b6deec4d2a19a4ec02a428ed797143262ba  guix-build-ef77549000ac/output/x86_64-linux-gnu/SHA256SUMS.part
cf87bbbb54d1fb85f21e02f3c99e0fc54a787a2b7aba1a248ea06af16ed26f57  guix-build-ef77549000ac/output/x86_64-linux-gnu/bitcoin-ef77549000ac-x86_64-linux-gnu-debug.tar.gz
088488e2ad215d5462ae65e48b14dd104fc9d8b46420aed045c2f86c63a15e28  guix-build-ef77549000ac/output/x86_64-linux-gnu/bitcoin-ef77549000ac-x86_64-linux-gnu.tar.gz
a489b8c89e2215ba3b747d6d46bccd84b98e99b0fedcae717eb4d1e83c5a1c9d  guix-build-ef77549000ac/output/x86_64-w64-mingw32/SHA256SUMS.part
58307ce68de87d54df983e528df6c1ce38ed6245978b05a2517a5b06e4fb0455  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win-unsigned.tar.gz
48486afba892e0cb5995110e206caaddb16e920af9333c8d9e9bb72687674c01  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64-debug.zip
c408ec8aa64d637d295c852aefb572a148edcf997dba7c452bf0ee0bc3ce4d33  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64-setup-unsigned.exe
431a7d8746904a076e851ccb6a82831a350420c24376c5ea0ac2aca7a30685ca  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64.zip
```

fanquake reviewed (approved):
ACK ef77549000ac46eededa3e4a89466024db111524

Guix build:
```bash
3895e9c0877221d48d0c4c601a70f8568656190a98bc16852484c255909db98a  guix-build-ef77549000ac/output/aarch64-linux-gnu/SHA256SUMS.part
e08470526151885152d45f521309d50049639ff350ffbc6f3509ea86be0e391c  guix-build-ef77549000ac/output/aarch64-linux-gnu/bitcoin-ef77549000ac-aarch64-linux-gnu-debug.tar.gz
4a4f0ff43840acf9e563086c368f60cf7ead50d91357b0a6474cbfe5954870b4  guix-build-ef77549000ac/output/aarch64-linux-gnu/bitcoin-ef77549000ac-aarch64-linux-gnu.tar.gz
16a0145e0e787f16f26e68320074a52045338a90ee12ec83aaca6a99c0eb7681  guix-build-ef77549000ac/output/arm-linux-gnueabihf/SHA256SUMS.part
10a7aef5accffa67829ecd015ec8f52aa5907641cf4d60bcf29fd20e93418227  guix-build-ef77549000ac/output/arm-linux-gnueabihf/bitcoin-ef77549000ac-arm-linux-gnueabihf-debug.tar.gz
8ced201f710f4913fdff36061dbe01c7fc64a0dc97f92c00b22e512c09125fd9  guix-build-ef77549000ac/output/arm-linux-gnueabihf/bitcoin-ef77549000ac-arm-linux-gnueabihf.tar.gz
d50770610865e6978efc6e6c60e81358b88bf560872ba30bbe9c200cf2c4c618  guix-build-ef77549000ac/output/dist-archive/bitcoin-ef77549000ac.tar.gz
2fed92dc00eb5a96817ca955171fb1b9d18fe8d9ad67999568f1eac4e1e16279  guix-build-ef77549000ac/output/powerpc64-linux-gnu/SHA256SUMS.part
80b16d5cf1d4f5ee4aa8072b7c003f412236a0517feee5933afc768a8b21d4c2  guix-build-ef77549000ac/output/powerpc64-linux-gnu/bitcoin-ef77549000ac-powerpc64-linux-gnu-debug.tar.gz
b15ffbd4533bbc4f9411b749e0ca3938318369c4cfc61d7d28d03b09b3ac7373  guix-build-ef77549000ac/output/powerpc64-linux-gnu/bitcoin-ef77549000ac-powerpc64-linux-gnu.tar.gz
96665ee949e487170bf2cdb98caea3d530e8b922f9a9420ea222b0abda1a1754  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/SHA256SUMS.part
d7177210324ed92d6c35b615b8614b07b31f2a3ce543da55034e3bc680d24abb  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/bitcoin-ef77549000ac-powerpc64le-linux-gnu-debug.tar.gz
934cad3c42f1029e1f7f1f4b0a888b1d266ada67fa571b847a6d3c29ff894fa3  guix-build-ef77549000ac/output/powerpc64le-linux-gnu/bitcoin-ef77549000ac-powerpc64le-linux-gnu.tar.gz
28df25c8dcd46177da706c43ebf272201aec36fdd93cbd73fade71e52851140f  guix-build-ef77549000ac/output/riscv64-linux-gnu/SHA256SUMS.part
bc72438937da78740fbc364d4c1102b3821d2c686ba4bc188001fcf7a3ca2b98  guix-build-ef77549000ac/output/riscv64-linux-gnu/bitcoin-ef77549000ac-riscv64-linux-gnu-debug.tar.gz
fbadeffc83e3dbb31a702e1406616469c5ad84aa70a34ca5b908fa680d7348f4  guix-build-ef77549000ac/output/riscv64-linux-gnu/bitcoin-ef77549000ac-riscv64-linux-gnu.tar.gz
acfdefec7ac276ba7bac88982a4200dc976263831a8581c87c00eef1a87a6c4f  guix-build-ef77549000ac/output/x86_64-apple-darwin/SHA256SUMS.part
237d290ef473736bf22e23f29fe277770497c39c9e36908b2816e3ce8dad2822  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx-unsigned.dmg
2bb6412ab793538f39aa73d5bb868a6b85da85d387eb672652d88ae7d7b18d26  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx-unsigned.tar.gz
605402a386e4444fe31fa70cc79096e4da446310616ad5b108afebfdec43fbf2  guix-build-ef77549000ac/output/x86_64-apple-darwin/bitcoin-ef77549000ac-osx64.tar.gz
a5e752f3b796841a2edac7621c290b6deec4d2a19a4ec02a428ed797143262ba  guix-build-ef77549000ac/output/x86_64-linux-gnu/SHA256SUMS.part
cf87bbbb54d1fb85f21e02f3c99e0fc54a787a2b7aba1a248ea06af16ed26f57  guix-build-ef77549000ac/output/x86_64-linux-gnu/bitcoin-ef77549000ac-x86_64-linux-gnu-debug.tar.gz
088488e2ad215d5462ae65e48b14dd104fc9d8b46420aed045c2f86c63a15e28  guix-build-ef77549000ac/output/x86_64-linux-gnu/bitcoin-ef77549000ac-x86_64-linux-gnu.tar.gz
a489b8c89e2215ba3b747d6d46bccd84b98e99b0fedcae717eb4d1e83c5a1c9d  guix-build-ef77549000ac/output/x86_64-w64-mingw32/SHA256SUMS.part
58307ce68de87d54df983e528df6c1ce38ed6245978b05a2517a5b06e4fb0455  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win-unsigned.tar.gz
48486afba892e0cb5995110e206caaddb16e920af9333c8d9e9bb72687674c01  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64-debug.zip
c408ec8aa64d637d295c852aefb572a148edcf997dba7c452bf0ee0bc3ce4d33  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64-setup-unsigned.exe
431a7d8746904a076e851ccb6a82831a350420c24376c5ea0ac2aca7a30685ca  guix-build-ef77549000ac/output/x86_64-w64-mingw32/bitcoin-ef77549000ac-win64.zip
```

hebasto commented:
> Unfortunately, our depends build system still lacks 100% reliability.

The depends build system improvements are continued in #22552.

:tiger2:
